src/cases/schemas/cases/pages/pages.schema.js

```diff
@@ -0,0 +1,48 @@
+import Joi from "joi";
+
+const fieldSchema = Joi.object({
+  ref: Joi.string().required(),
+  type: Joi.string().valid("string", "number", "boolean", "date").required(),
+  label: Joi.string().required(),
+  format: Joi.string().optional(),
+});
+
+const sectionSchema = Joi.object({
+  title: Joi.string().required(),
+  type: Joi.string().valid("object", "array").required(),
+  component: Joi.string().valid("list", "table").required(),
+  fields: Joi.array().items(fieldSchema).required(),
+});
+
+const tabSchema = Joi.object({
+  title: Joi.string().required(),
+  sections: Joi.array().items(sectionSchema).required(),
+});
+
+const bannerSummarySchema = Joi.object().pattern(
+  Joi.string(),
+  Joi.object({
+    label: Joi.string().required(),
+    ref: Joi.string().required(),
+    type: Joi.string().valid("string", "number", "boolean", "date").required(),
+  }),
+);
+
+const bannerSchema = Joi.object({
+  title: Joi.object({
+    ref: Joi.string().required(),
+    type: Joi.string().valid("string", "number", "boolean", "date").required(),
+  }).required(),
+  summary: bannerSummarySchema.required(),
+});
+
+const detailsSchema = Joi.object({
+  banner: bannerSchema.required(),
+  tabs: Joi.object().pattern(Joi.string(), tabSchema).required(),
+});
+
+const pagesSchema = Joi.object({
+  details: detailsSchema.required(),
+});
+
+export { pagesSchema };
```

Comment:
There is already a schema used to validate updates to the workflow, could it be used? https://github.com/DEFRA/fg-cw-backend/blob/fbe35f8f6732a23b79ff620fd677bfc300392797/src/cases/schemas/workflow.schema.js#L50-L52

The existing schemas should really have been broken out into their own files as it's difficult to spot otherwise. Maybe it's a good time to do as it's the first time they are reused?